src/error.rs

```diff
@@ -141,8 +144,10 @@ impl WebDriverError {
 impl ToJson for WebDriverError {
     fn to_json(&self) -> Json {
         let mut data = BTreeMap::new();
-        data.insert("error".to_string(), self.status_code().to_json());
-        data.insert("message".to_string(), self.message.to_json());
+        data.insert("error".into(), self.status_code().to_json());
+        data.insert("message".into(), self.message.to_json());
+        data.insert("stacktrace".into(),
+                    format!("{:?}", self.backtrace).to_json());
```

Comment:
I couldn’t find a better way to do this. Unfortunately it is not allowed to implement traits for generic types in other crates. )-: